extra/1-currency-conversion.js

```diff
@@ -5,7 +5,9 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(num) {
```

Comment:
Nice function, but could you be clearer with your naming of num?

Reply:
changed it for a price, thx